src/content/api/module-variables.mdx

```diff
@@ -105,6 +105,22 @@ console.log(import.meta.webpack); // output `5` for webpack 5
 
 Webpack specific. An alias for [`module.hot`](#modulehot-webpack-specific), however `import.meta.webpackHot` can be used in [strict ESM](/guides/ecma-script-modules/#flagging-modules-as-esm) while `module.hot` can't.
 
+## import.meta.webpackContext
+
+Returns same as `require.context` but only for `javascript/auto` and `javascript/esm`.
+
+- Type: `function`
```

Comment:
Let's document the whole type.

```suggestion
- Type:
  ```ts
  (
		request: string,
		options?: {
			recursive?: boolean;
			regExp?: RegExp;
			include?: RegExp;
			exclude?: RegExp;
			preload?: boolean | number;
			prefetch?: boolean | number;
			chunkName?: string;
			exports?: string | string[][];
			mode?: "sync" | "eager" | "weak" | "lazy" | "lazy-once";
		}
	) => webpack.Context
```